test/unit-test/AuthenticatorTests.cpp

```diff
@@ -144,49 +144,6 @@ BOOST_AUTO_TEST_CASE(Given_GoodToken_When_Validate_Shall_ValidateTokenSuccessful
   BOOST_TEST(res == std::chrono::time_point_cast<std::chrono::seconds>(exprTime).time_since_epoch().count());
 }
 
-BOOST_AUTO_TEST_CASE(Given_BadPathInToken_When_Validate_Shall_ThrowException)
```

Comment:
why remove this unittest?

Reply:
Because this one does not make sense, I couldn't gather the intention. It claims it is about "Bad path in tokens", but the paths are ok, and anyway, I think there is no such thing as a bad path in a token.